### DISCUSSION TITLE
直播弹幕发送栏相关问题

### DISCUSSION THREAD

**Discussion by qiuyue0** [the1812/Bilibili-Evolved#3475] [提问 / Q&A] [answered]
使用直播弹幕发送栏发送弹幕，打字的时候有概率突然隐藏，使用enter将输入法编码内容上屏的时候会直接发送当前已输入内容并上屏两次编码。

下图gif展示了上述两个问题
![test](https://user-images.githubusercontent.com/18420831/176994284-58370a49-2e56-4392-8126-1668c4b9deab.gif)

我当前使用的是多多输入法，为了排除输入法问题，使用手心输入法和搜狗输入法进行测试，得到了相同的结果。需要注意的是，网页全屏不会出现该问题。

由于手上没有其他设备，所以我想知道这个问题是由设备引起的，还是脚本bug。

**Comment by the1812** [comment, marked as answer]
我那个描述忘更新了, 只对网页全屏有效, 全屏的这个是 b 站自带的
